src/content/blog/orbit-year-in-review-growing-a-global-community-of-builders.md

```diff
@@ -5,7 +5,7 @@ created-on: 2023-12-07T13:50:04.187Z
 updated-on: 2023-12-07T13:50:04.200Z
 published-on: 2023-12-07T13:50:04.211Z
 featured: false
-category: announcements
+category: Announcements
```

Comment:
Are we capitalizing this for styling purposes?

Reply:
Yes, could do it in the code if you think makes more sense?

Reply:
Yes - It's easier to standardize this in the code then having to make sure the data is always correctly formated.